change/@fluentui-react-carousel-2e2a03df-5a80-41c4-b8df-9745b659c556.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33155&commitId=aa5a7dbb8d17554a3e40f1786e8b224537ddc4af&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|21| Changed|
|Avatar Converged.badgeMask.chromium.png|30| Changed|

</details>


  <details><summary><b>Drawer</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay High Contrast.chromium.png|4082| Changed|
|Drawer.overlay drawer full.chromium.png|1154| Changed|
|Drawer.Full Overlay Dark Mode.chromium.png|8| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>